docs/standard/ipn.rst

```diff
@@ -39,7 +46,7 @@ Using PayPal Standard IPN
 
            # What you want the button to do.
            paypal_dict = {
-               "business": settings.PAYPAL_RECEIVER_EMAIL,
+               "business": "receiver_email@example.com",
```

Comment:
We also need to update the example code under section 5, `show_me_the_money` to check the receiver email matches "receiver_email@example.com" at the beginning, and return if it doesn't match.